docs/reference/buildx_history_inspect.md

```diff
@@ -21,8 +21,20 @@ Inspect a build
 
 <!---MARKER_GEN_END-->
 
+## Description
+
+Inspect a build record to view metadata such as duration, status, build inputs,
+platforms, outputs, and attached artifacts. You can also use flags to extract
+provenance, SBOMs, or other detailed information.
+
 ## Examples
 
+### Inspect a build and print metadata
+
+```console
+docker buildx history inspect mybuild
```

Comment:
Same a previous comment, it should be a build ref.

Also like the `history logs` command example we could also say that `history inspect` inspects the most recent build. Same for `export`.